manifests/plugin/openscap.pp

```diff
@@ -0,0 +1,31 @@
+# = Foreman Proxy OpenSCAP plugin
+#
+# This class installs OpenSCAP plugin
+#
+# === Parameters:
+#
+# $enabled::                    enables/disables the plugin
+#                               type:boolean
+# $openscap_send_log_file::     Log file for the forwarding script
+#
+# $spooldir::                   Directory where OpenSCAP audits are stored
+#                               before they are forwarded to Foreman
+class foreman_proxy::plugin::openscap (
+  $enabled                = $::foreman_proxy::plugin::openscap::params::enabled,
+  $openscap_send_log_file = $::foreman_proxy::plugin::openscap::params::openscap_send_log_file,
+  $spooldir               = $::foreman_proxy::plugin::openscap::params::spooldir,
+) inherits foreman_proxy::plugin::openscap::params {
+  validate_bool($enabled)
+
+  case $::osfamily {
```

Comment:
That sounds unlikely to me.. often people misinterpret the logs:

<pre>
# puppet apply -e "package { 'abcdef': } -> file { '/tmp/proxy.conf': ensure => present }"
Notice: Compiled catalog for foreman.example.com in environment production in 0.47 seconds
Error: Execution of '/bin/yum -d 0 -e 0 -y list abcdef' returned 1: Error: No matching Packages to list
Error: /Stage[main]/Main/Package[abcdef]/ensure: change from absent to present failed: Execution of '/bin/yum -d 0 -e 0 -y list abcdef' returned 1: Error: No matching Packages to list
Notice: /Stage[main]/Main/File[/tmp/proxy.conf]: Dependency Package[abcdef] has failures: true
Warning: /Stage[main]/Main/File[/tmp/proxy.conf]: Skipping because of failed dependencies
Notice: Finished catalog run in 0.87 seconds
</pre>

The bits of about dependency failures are it actually not evaluating the file resource.. noisily.